### PR TITLE
Add mSOL and stSOL swtichboard v2 oracles

### PR DESCRIPTION
**Pull request by elliotkennedy**
- USDT/USD (bound) - https://switchboard.xyz/explorer/3/8KCm1o99ZvhZBiFfsDpSK4EC6iDeqS5NkxQtaCigQTqB
- USDC/USD (bound) - https://switchboard.xyz/explorer/3/DZxQcTQnNMskQ2VEX57jNW1rBpLTVEVcDYySFkhDaLCe
- SOL/USD - https://switchboard.xyz/explorer/3/4aUbjcFqQNxMVjsgnuxAAmtPZHyyZBsLeJjBM8LYrkFX
- mSOL/USD (bound) - https://switchboard.xyz/explorer/3/856a1KbnQa7gWzaqmjaYWWAGqcqegZxmaatRig3hzfv9
- stSOL/USD (bound) - https://switchboard.xyz/explorer/3/AMtCwAExNkNiFFMF7QCDA2RqbWi816K7woQq8KB6emHj

### CODE REVIEW

File: third-party/switchboard/switchboard-test.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-curl https://api.switchboard.xyz/api/test -X POST -H "Content-Type: application/json" -d "@$1"
\ No newline at end of file
+curl https://api.switchboard.xyz/api/test -X POST -H "Content-Type: application/json" -d "@$1" | python -m json.tool
```
Comment (oeble) [inline third-party/switchboard/switchboard-test.sh:3]: What `json.tool` does?

File: third-party/switchboard/feeds/msol-usd.json [hunk 2]
```diff
@@ -0,0 +1,419 @@
...
+            "upperBound": {
+              "tasks": [
+                {
+                  "valueTask": {
+                    "big": "${SOL_USD}"
+                  }
+                },
+                {
+                  "multiplyTask": {
+                    "scalar": 1.2
```
Comment (oeble) [inline third-party/switchboard/feeds/msol-usd.json:409]: We need a remind me to increase this in the future

**Review by oeble — APPROVED** [review]
LGTM

File: third-party/switchboard/switchboard-test.sh (hunk 1 above)
Comment (elliotkennedy) [reply third-party/switchboard/switchboard-test.sh:3]: Pretty prints json - it's on most unix boxes

File: third-party/switchboard/feeds/msol-usd.json (hunk 2 above)
Comment (elliotkennedy) [reply third-party/switchboard/feeds/msol-usd.json:409]: Will add to weekly

Comment (elliotkennedy) [reply third-party/switchboard/feeds/msol-usd.json:409]: https://github.com/hubbleprotocol/weekly/commit/bca33708fcb99d3fffb444d90761ea95a9c5d844

Comment (elliotkennedy) [reply third-party/switchboard/feeds/msol-usd.json:409]: I can do an automated alert for this later too

**Pull request merged**